acrobat/blocks/dc-converter-widget/dc-converter-widget.js

```diff
@@ -113,7 +113,21 @@ const verbRedirMap = {
   'number-pages': 'number',
 };
 
+const exhCreateLimitCookie = 'cr_p_c_e';
+const exhExportLimitCookie = 'ex_p_c_e';
+const exhOrganizeLimitCookie = 'or_p_c_e';
+const exhCompressLimitCookie = 'cm_p_ops_e';
+
+const exhLimitCookieMap = {
+  'to-pdf': exhCreateLimitCookie,
+  'pdf-to': exhExportLimitCookie,
+  'compress-pdf': exhCompressLimitCookie,
+  'rotate-pages': exhOrganizeLimitCookie,
+};
+
 const url = window.location;
+const cookies = document.cookie;
```

Comment:
```suggestion
const { cookie } = document;
```
and change cookies -> cookie